packages/light-apps/src/IdentityHeader/IdentityHeader.tsx

```diff
@@ -147,12 +138,11 @@ export class IdentityHeader extends React.PureComponent<Props, State> {
 
   handleChangeCurrentAccount = (account: string) => {
     const { history } = this.props;
-    const currentLocation = this.getCurrentLocation();
 
-    history.push(`/${currentLocation}/${account}`);
+    history.push(`/identity/${account}`);
   }
 
-  handleToggleApp = (event: React.SyntheticEvent<HTMLElement, Event>, { value }: DropdownProps) => {
+  handleToggleApp = (_event: React.SyntheticEvent<HTMLElement, Event>, { value }: DropdownProps) => {
```

Comment:
question: what is the standard/house rules for using underscore param name? so I can be sure next time

Reply:
in rust _variable means the variable is unused. In TS too, it shows me the variable with some opacity in VSCode (meaning unused), unless I add the _ in front